src/Symfony/Bridge/Monolog/Processor/AbstractTokenProcessor.php

```diff
@@ -19,9 +20,13 @@
  *
  * @author Dany Maillard <danymaillard93b@gmail.com>
  * @author Igor Timoshenko <igor.timoshenko@i.ua>
+ *
+ * @internal since Symfony 6.1
```

Comment:
shouldn't be internal as this abstract class is meant to be used also by userland to me

Reply:
Can we finalize the `__invoke()` method instead? It is the implementation of an external contract and not being able to change its signature was one of the challanges we've faced here.

Reply:
if CompatibilityProcessor is not internal as I suggested, then ppl should use it and implement doInvoke instead,
this would solve the issue without requiring final/internal